content/modules/modules-quickstart.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Modules Quick Start
```

Comment:
No pictures here?

Reply:
Do you think we need them? Seems pretty straightforward to me.

Reply:
I think we need them.